features/admin.groups.v1/pages/groups.tsx

```diff
@@ -187,41 +186,39 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
             value: ""
         };
 
-        if (isSuperOrganization() || isFirstLevelOrganization()) {
-            setUserStoresListRequestLoading(true);
-            getUserStoreList()
-                .then((response: AxiosResponse<UserstoreListResponseInterface[]>) => {
-                    if (storeOptions?.length === 0) {
-                        storeOptions.push(storeOption);
-                    }
-
-                    response.data.map((store: UserstoreListResponseInterface, index: number) => {
-                        if (store.name.toUpperCase() !== userstoresConfig.primaryUserstoreName) {
-                            setUserStoreRequestLoading(true);
-                            getAUserStore(store.id).then((response: UserStorePostData) => {
-                                const isDisabled: boolean = response.properties.find(
-                                    (property: UserStoreProperty) => property.name === "Disabled")?.value === "true";
-
-                                if (!isDisabled) {
-                                    storeOption = {
-                                        disabled: store.typeName === RemoteUserStoreManagerType.RemoteUserStoreManager,
-                                        key: index,
-                                        text: store.name,
-                                        value: store.name
-                                    };
-                                    storeOptions.push(storeOption);
-                                }
-                            }).finally(() => {
-                                setUserStoreRequestLoading(false);
-                            });
-                        }
-                    });
+        setUserStoresListRequestLoading(true);
+        getUserStoreList()
+            .then((response: AxiosResponse<UserstoreListResponseInterface[]>) => {
+                if (storeOptions?.length === 0) {
+                    storeOptions.push(storeOption);
+                }
 
-                    setUserStoresList(storeOptions);
-                }).finally(() => {
-                    setUserStoresListRequestLoading(false);
+                response.data.map((store: UserstoreListResponseInterface, index: number) => {
+                    if (store.name.toUpperCase() !== userstoresConfig.primaryUserstoreName) {
+                        setUserStoreRequestLoading(true);
+                        getAUserStore(store.id).then((response: UserStorePostData) => {
```

Comment:
Do we need to get each userstore individually through API and check for the disabled status? AFAIR, the disabled status comes in the `/userstores` API response.

Reply:
Resolving as per the offline discussion since this is existing code